creusot-contracts/src/resolve.rs

```diff
@@ -29,7 +29,7 @@ pub fn structural_resolve<T: ?Sized>(_: T) -> bool {
     dead
 }
 
-impl<T1, T2: ?Sized> Resolve for (T1, T2) {
+impl<T1, T2> Resolve for (T1, T2) {
```

Comment:
Why is this needed?

Reply:
There was a bug in the proc macro where field projections were wrapped in `{ ... }`, which must be `Sized`.

Desugaring of `resolve(self.1)` before:
```
resolve({ (*&self).1 })
```

I've fixed it:
```
resolve((*&self).1)
```